src/torchcodec/decoders/_audio_decoder.py

```diff
@@ -75,6 +75,14 @@ def __init__(
             sample_rate if sample_rate is not None else self.metadata.sample_rate
         )
 
+    def get_all_samples(self) -> AudioSamples:
+        """Returns all the audio samples from the source.
+
+        Returns:
+            AudioSamples: The samples within the file.
+        """
+        return self.get_samples_played_in_range()
```

Comment:
Can we add a brief comment explaining that the defaults for `get_samples_played_in_range()` do what we want, and that this method is a way of advertising that to folks?

Reply:
Absolutely, do you want it as a comment or in the docstring?